modules/markup/html_test.go

```diff
@@ -342,6 +342,7 @@ func TestRegExp_IssueNumericPattern(t *testing.T) {
 		"#1234",
 		"#0",
 		"#1234567890987654321",
+		"[#1234]",
 	}
 	falseTestCases := []string{
 		"# 1234",
```

Comment:
Just to make sure I would add `[]` `[X]` that are checklist element even if there should be replace before by markdown.

Reply:
Added the test cases